documentation/src/main/asciidoc/topics/dependencies/hibernate_v5.2.adoc

```diff
@@ -0,0 +1,8 @@
+[source,xml,options="nowrap"]
+----
+<dependency>
+   <groupId>org.infinispan</groupId>
+   <artifactId>infinispan-hibernate-cache</artifactId>
+   <version>${version.infinispan}</version> <!-- 9.2.x.Final -->
```

Comment:
is this still true @galderz ? infinispan version

Reply:
@karesti I checked with @rvansa here. hibernate v5.2 is supported with 9.2.x only.